src/panels/lovelace/editor/card-editor/hui-dialog-delete-card.ts

```diff
@@ -83,6 +83,7 @@ export class HuiDialogDeleteCard extends LitElement {
   private _close(): void {
```

Comment:
Make this function public, so closing with the back button also works

Reply:
You mean the browser back button? If yes, this does not work even if I make the method public. Why/how should that work?